### DISCUSSION TITLE
Exploring New Frameworks for Frontend Development

### DISCUSSION THREAD

**Discussion by kingtops** [community/community#107622] [Programming Help] [answered]
### Body

Hey everyone! 👋 We're excited to dive into the world of frontend development and explore some new frameworks that have been gaining traction lately. Whether you're a seasoned developer or just starting out, this discussion is for you! Let's share our experiences, pros and cons, and any tips or resources we've found helpful. Whether it's React, Vue.js, Angular, or something else entirely, let's discuss what's been working well for us and what we're excited to try next. Can't wait to hear from you all!

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by kingtops** [comment]
Has anyone tried out Svelte for frontend development? I've been hearing a lot about it lately and I'm curious to hear people's experiences.

**Comment by mightstar** [comment]
I've played around with Svelte a bit and found it really interesting! The way it compiles components down to vanilla JS is pretty neat.

**Comment by kingtops** [comment]
That's great to hear! I've been thinking about giving it a try for an upcoming project. Any tips for getting started?

**Comment by mightstar** [comment, marked as answer]
Also, I've been using React for most of my projects. It seems like it could be a good alternative for certain types of applications.
